nix/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, python311Packages
+, pkgs
+, fetchFromGitHub
+}:
+
+let
+  # Define the Python packages required
+  pythonPackages = pkgs.python311.withPackages (ps: with ps; [
```

Comment:
@scientiac Could be source file the same `requirements.txt`? It is duplicated information now and `xkbcommon<1.1` has mentioned version originally.

Reply:
Is there anything that is breaking on the version i have used? I couldnt find the older version packaged on the nixpkgs repo. These packages call from the nixpkgs repo and not the default python repo where pip fetches it's packages from.

Reply:
@scientiac Unfortunatelly the version of xkcommon, <1.1 has to be explicitly defined like this. Otherwise, it will not run under x11. More [here](https://github.com/asus-linux-drivers/asus-numberpad-driver/issues/186#issuecomment-2241063958).